api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -102,14 +102,31 @@ public void updateCompletionTime(
     userAccessModuleDao.save(userAccessModuleToUpdate.setCompletionTime(timestamp));
   }
 
+  private Optional<AccessModuleStatus> mapToEnabledAccessModule(
```

Comment:
nit: I think usually non-visible members should below visible members. 
some references saw before: go/java-practices/class-structure#general

Reply:
agreed, thanks. Done.